[26.04] Docs: ROCm-RAG update (#9)
ROCm/rocm-rag: docs/how-to/test-pipelines.rst

> Test the RAG pipelines with Open-WebUI

Reviewer: Please confirm name - the docs show it as this:

<img width="2243" height="382" alt="Image" src="https://github.com/user-attachments/assets/8d93e5b2-7426-4c26-9835-6fe0200177bb" />

**************************************

After you've executed the ROCm-RAG pipelines, you can test them with `Open-WebUI <https://github.com/open-webui/open-webui>`__. When your retrieval pipeline is up and running (make sure all components are ready by checking the logs), you can access the Open-WebUI frontend by navigating to ``https://<Your deploy machine IP>`` or ``http://<Your deploy machine IP>:8080``.   

When you set up a new Open-WebUI account, your user data is saved to ``/rag-workspace/rocm-rag/external/open-webui/backend/data``.  

Set up and test pipelines with Open-WebUI
==========================================

> 1. Go to the admin panel.

Reviewer: ```suggestion
1. Go to the Admin panel.
```

   **Available models**

   Here's a list of models provided by these APIs. This is retrieved by calling ``http GET API_URL:PORT/v1/models``:  

   .. image:: ../images/admin_list_of_models.png
      :align: center
      :alt: List of available models

   **Model settings**

   By default, models are only accessible by admin. Make sure you share the model to public (all registered users) or private groups:  

   .. image:: ../images/admin_model_settings.png
      :align: center
      :alt: Model settings

5. Configure audio settings.

   Select which fast whisper model (here's a `list of models provided by fast-whisper <https://github.com/SYSTRAN/faster-whisper/blob/d3bfd0a305eb9d97c08047c82149c1998cc90fcb/faster_whisper/utils.py#L12>`__) and which TTS voice to use:   

   .. image:: ../images/admin_audio.png
      :align: center
      :alt: Audio settings

Compare RAG pipeline against direct LLM responses
==================================================

You can test the accuracy of the ROCm-RAG pipelines through direct LLM responses. 

Test direct LLM response
-------------------------

> As an example, select **Qwen/Qwen3-30B-A3B-Instruct-2507** from the model drop-down and ask: ``What is ROCm-DS?``. Since ROCm-DS is a newer GPU-accelerated data science toolkit by AMD, the direct LLM output may contain hallucinations:

Reviewer: ```suggestion
As an example, select **Qwen/Qwen3-30B-A3B-Instruct-2507** from the model drop-down and ask: ``What is ROCm-DS?``. Because ROCm-DS is a newer GPU-accelerated data science toolkit by AMD, the direct LLM output might contain hallucinations:
```
